tests/php81_enums.phpt

```diff
@@ -0,0 +1,145 @@
+--TEST--
+Enums in php 8.1
+--SKIPIF--
+<?php if (PHP_VERSION_ID < 80100) die('skip PHP >= 8.1 only'); ?>
+--FILE--
+<?php
+
+require __DIR__ . '/../util.php';
+
+$code = <<<'PHP'
+<?php
+enum HasValue: int {
+    case FOO = 42;
+    const FOO_ALIAS = self::FOO;
+}
+
+enum NoValue {
+    case FOO;
+}
+PHP;
+
+$node = ast\parse_code($code, $version=70);
+echo ast_dump($node), "\n";
+$node = ast\parse_code($code, $version=80);
+echo ast_dump($node), "\n";
+$node = ast\parse_code($code, $version=85);
+echo ast_dump($node), "\n";
+--EXPECTF--
+AST_STMT_LIST
+    0: AST_CLASS
+        flags: CLASS_FINAL | CLASS_ENUM (%d)
+        name: "HasValue"
+        docComment: null
+        extends: null
+        implements: null
+        stmts: AST_STMT_LIST
+            0: AST_ENUM_CASE
+                name: "FOO"
+                expr: 42
+                0: null
```

Comment:
What's that zero index doing here?

Reply:
Missed that - changed to attributes

I started writing this a month ago and didn't notice the 0. Enum cases can have attributes - I updated the test and documentation.

Because this is a brand new node kind, it shouldn't be a breaking change to always set attributes in `$node->children` instead of an instance property